src/pages/home/report/ReportActionsView.js

```diff
@@ -99,6 +99,7 @@ class ReportActionsView extends React.Component {
         this.state = {
             isLoadingMoreChats: false,
             isMarkerActive: false,
+            unreadActionCount: this.props.report.unreadActionCount,
```

Comment:
I don't think it's great that this state variable shadows the name of the prop. Instead, we should call it `localUnreadActionCount` or something. And then `updateUnreadMessageCount` would be `updateLocalUnreadActionCount`